frontend/src/pages/SketchListPage.tsx

```diff
@@ -18,6 +18,14 @@ export function SketchListPage() {
   }, [dispatch]);
 
   if (!sketches) return <p>Loading...</p>;
+  if (sketches.length === 0) {
+    return (
+      <div className="mt-[-20px] flex h-screen-without-header flex-col items-center justify-center space-y-4">
+        <img src="./alert-circle.svg" alt="alert" className="mx-auto mt-20 size-20" />
+        <div className="text-4xl text-gray-200">まだ投稿がありません</div>
+      </div>
```

Comment:
[imo]
ここ3点確認お願いシャス！

1. アイコン部分はreact-featherにコンポーネントが用意されてて、それを使った方がスマートだと思う！
2. 「まだ投稿がありません」は`font-semibold`でお願い 🙏 
3. 外側のdivに付いてる`mt-[-20px]`で何かしらスペース調整してると思うんだけど、これって何を考慮してるって感じかな？

結論私なら↓のように実装すると思う！
```suggestion
      <div className="flex h-screen-without-header flex-col items-center justify-center space-y-4 text-gray-200">
        <AlertCircle size={80} />
        <p className="text-4xl font-semibold">まだ投稿がありません</p>
      </div>
```

Reply:
修正した